hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/AbstractClientCacheProxyBase.java

```diff
@@ -165,13 +180,24 @@ public void open() {
     }
 
     protected abstract void closeListeners();
-    //endregion close&destroy
 
-    //region DISTRIBUTED OBJECT
-    public String getNameWithPrefix() {
+    @Override
+    public String getPrefixedName() {
         return nameWithPrefix;
     }
 
+    /**
+     * Gets the full cache name with prefixes.
+     *
+     * @return the full cache name with prefixes
+     *
+     * @deprecated use #getPrefixedName instead
+     */
+    @Deprecated
```

Comment:
This method was public so instead of removing it silently, I have just added `@Deprecated` on it. It is not used by our code base at the moment, but I have thought that there might be someone uses this public method then instead of removing it, I have just deprecated it.